test/testinput/bufr_airs_new_format.yaml

```diff
@@ -0,0 +1,149 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "/data/users/eliu/bufr_samples/gdas.t00z.airsev.tm00.bufr_d"
+      mnemonicSets:
+        - mnemonics: [SAID, SIID, FOVN, YEAR, MNTH, DAYS, HOUR, MINU, SECO, CLAT, CLON, CLATH, CLONH] 
+        - mnemonics: [LSQL, HOLS, HMSL]
+        - mnemonics: [SAZA, SOZA, BEARAZ, SOLAZI]
+        - mnemonics: [TMBR]
+          channels: 1-10
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: YEAR
+              month: MNTH
+              day: DAYS
+              hour: HOUR
+              minute: MINU
+              second: SECO
+          height: 
+            mnemonic: HMSL
+          hols: 
```

Comment:
Please avoid using abbreviations as keys, I would use names that are easily understandable. I would use the same name you use when you define the ioda object (heightOfSurface in this case). Part of the intent here is to leave the BUFR conventions and names behind (they are isolated in the BUFR section).

Reply:
Agreed. I just changed all keys to avoid using of abbreviations.